plaidml/bridge/openvino/ops/psroi_pooling.cpp

```diff
@@ -0,0 +1,256 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset5.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using namespace edsl;             // NOLINT[build/namespaces]
+
+namespace {
+
+template <typename T>
+std::vector<T> cast_constant_operand(size_t operand_idx, ngraph::Node* layer) {
```

Comment:
Can you use the function here https://github.com/Flex-plaidml-team/plaidml/blob/plaidml-v1/plaidml/bridge/openvino/plaidml_util.hpp#L28 ?

Reply:
yes, it can work correctly by use the function in the above link, i will remove the cast_constant_operand function.